tests/reliability/input_formats/generated/test_markdown_1/program.py

```diff
@@ -0,0 +1,49 @@
+### Input models ###
+
+
+from pydantic import BaseModel, Field
+
+
+class ProgramInputs(BaseModel):
+    markdown_content: str = Field(
+        ...,
+        description="The content of the markdown document from which the table of contents will be generated.",
+    )
+
+
+### Output models ###
+
+
+from pydantic import BaseModel, Field
+
+
+class ProgramOutputs(BaseModel):
+    table_of_contents: str = Field(..., description="The generated table of contents in markdown format.")
+
+
+### Program definition ###
+
+import dspy
+
+
+class BaseSignature(dspy.Signature):
```

Comment:
Would this test have been more useful / substantial if it asked to produce a nested list or something? Markdown-to-Markdown is also interesting so it's fine to merge.